packages/components/CHANGELOG.md

```diff
@@ -4,14 +4,30 @@
 
 #### Breaking Changes
 
+- Components with login feature accept connectors instead of booleans [#90](https://github.com/liteflow-labs/liteflow-js/pull/90)
+- Components requiring the user account now require an optional `signer` in props [#90](https://github.com/liteflow-labs/liteflow-js/pull/90)
+- Pass `connector` to wallet components [#90](https://github.com/liteflow-labs/liteflow-js/pull/90)
+- Wallet components accept the `activate` function instead of the `onAuthenticated` callback [#90](https://github.com/liteflow-labs/liteflow-js/pull/90)
+- Remove providers from `LiteflowNFTApp` components [#90](https://github.com/liteflow-labs/liteflow-js/pull/90):
+  - `ApolloProvider` is removed and needs to be initialized by the application
+  - `Web3ReactProvider` is removed and needs to be initialized by the application
+  - `CookiesProvider` is removed and needs to be initialized by the application
+  - `LiteflowProvider` is removed and needs to be initialized by the application
+  - `Session` is removed and the session handling should be handled by the application
+
```

Comment:
What is the reason that you haven't also removed the Bugsnag `ErrorBoundary` from `LiteflowNFTApp`?

Reply:
Definitely a good point, I wanted to remove that as well but because it was not directly/indirectly related to the session removal I was thinking to do that on another PR